content/install/glitch.md

```diff
@@ -26,43 +21,11 @@ performed:
   credentials are safe from others. If you’re not able to do this, ask around
   in [Bastion HQ](https://discord.gg/fzx8fkt) and some amazing people
   will help you out**.
-* Click on the **Logs** button (at the top left corner) in your Glitch project.
-  It will open the log window at the bottom, where you can find the **Console**
-  button. Click on that and you’ll be redirected to your project’s console.
-* Now, before we install Bastion, we’ll need to clear all files from the default
-  project that was automatically created by Glitch. Run the following command in
-  your project’s console to clean everything up:
+* Click on the **Tools** button (at the bottom left corner) in your Glitch project.
+  Now click on the **Console**
+  button and you’ll be redirected to your project’s console.
+* To install Bastion just type this command then press enter
 
   ```text
-  rm -rf /app/* /app/.* 1>/dev/null
+  rm -rf /app/* /app/.* 1>/dev/null; git clone -b stable https://github.com/TheBastionBot/Bastion.git ~/; refresh
```

Comment:
See the earlier one, don't do it everything in one line. Explain what you're doing so the users who are reading it would understand the consequences of running a command. They shouldn't blindly run a command because it's written.